bot/exts/gurkcraft/gurkcraft.py

```diff
@@ -20,13 +27,57 @@ async def mcstatus(self, ctx: commands.Context) -> None:
             players = [user["name"] for user in status.raw["players"]["sample"]]
         except KeyError:
             players = ["None"]
+        except OSError:
+            await ctx.send("The server is currently offline.")
+            return
+
         embed = discord.Embed(title="Gurkcraft", color=Colours.green)
         embed.add_field(name="Server", value="mc.gurkult.com")
         embed.add_field(name="Server Latency", value=f"{status.latency}ms")
         embed.add_field(name="Gurkans Online", value=status.players.online)
         embed.add_field(name="Gurkans Connected", value=", ".join(players))
         await ctx.send(embed=embed)
 
+    @tasks.loop(minutes=2)
+    async def update_channel_description(self) -> None:
+        """Collect informations about the server and update the description of the channel."""
+        logger.debug("Updating topic of the #gurkcraft channel")
+
+        if not self.gurkcraft:
+            self.gurkcraft = await self.bot.fetch_channel(Channels.gurkcraft)
+
+            if not self.gurkcraft:
+                logger.warning(
+                    f"Failed to retrieve #gurkcraft channel {Channels.gurkcraft}. Aborting."
+                )
+                return
+
+        status = self.server.status()
+        is_online = True
+        try:
+            players = [user["name"] for user in status.raw["players"]["sample"]]
+        except KeyError:
+            players = []
+        except OSError:
+            is_online = False
+
+        if is_online:
+            description = (
+                "No players currently online. \n"
+                if not players
+                else (
+                    f"{status.players.online} player{'s' if len(players) > 1 else ''} "
+                    f"online: {', '.join(players)}. \n"
+                )
+                + f"Last server latency: {status.latency}ms"
+            )
+        else:
+            description = "The server is currently offline :("
+
+        await self.gurkcraft.edit(
+            topic=f"Our gurkan Minecraft server. Join: {Minecraft.server_address} ! \n{description}"
```

Comment:
```suggestion
            topic=f"Our gurkan Minecraft server. Join: {Minecraft.server_address}!\n{description}"
```
Assuming the "!" and " " don't serve any real function, other than looking nice, I think we should remove 'em. We could possibly get rid of the "!" if you think that would make things easier to parse (for the human eye).

Reply:
The space is important since the line break won't be shown on the main channel view, at the top. Otherwise, there would be no break between the two sentences, making it hard to read. We could replace it with a simple dot, but I personally like exclamation marks :P don't mind either way though.

Reply:
Sounds good. Feel free to do this however you want then :) 👍